combiner/allwpilib/combine.gradle

```diff
@@ -63,8 +63,6 @@ for (File it : allFiles) {
     continue
   }
 
-  def generateAll = matcher.group(1) == '_M_'

```

Comment:
Should the regex be updated to remove the `_M_` match as well?

Reply:
Removed and the group numbers were shifted down by one.